src/Symfony/Component/Serializer/Tests/Normalizer/AbstractObjectNormalizerTest.php

```diff
@@ -444,6 +445,14 @@ public function testNormalizeEmptyObject()
         $normalizedData = $normalizer->normalize(new EmptyDummy(), 'any', ['preserve_empty_objects' => true]);
         $this->assertEquals(new \ArrayObject(), $normalizedData);
     }
+
+    public function testNormalizeWithIgnoreAttributeAndPrivateProperties()
```

Comment:
```suggestion
    public function testNormalizeWithIgnoreAnnotationAndPrivateProperties()
```
Sorry, I missed it on my first review

Reply:
Aha, it's ok, I missed that as well!